src/cljs/athens/parser.cljs

```diff
@@ -9,7 +9,7 @@
   (insta/parser
    "S = c | link | bref | hash
     <c> = #'(\\w|\\s)+'
-    link = <'[['> c <']]'>
+    link = <'[['> c <']]'>  | <'[['> S? link S? <']]'>
     hash = <'#'> c | <'#'> <'[['> c <']]'>
     bref = <'(('> c <'))'>
    "))
```

Comment:
@anshbansal If my pull request gets merged #27, you could move this file to `src/cljs/athens/parser.cljc` and the test below would succeed via the `script/test/all` (that runs `lein test`)